bindings/java/src/org/sleuthkit/datamodel/timeline/StandardEventType.java

```diff
@@ -41,7 +41,8 @@
 	}
 
 	@Override
-	public TimelineEvent.EventDescription getDescription(String fullDescriptionRaw, String medDescriptionRaw, String shortDescriptionRaw) {
+	public TimelineEvent.EventDescription parseDescription(String fullDescriptionRaw, String medDescriptionRaw, String shortDescriptionRaw) {
```

Comment:
changed the name to parseDescription since it can involve inspection of the descriptions to reformulate them into multiple levels of detail.